src/dev/build/tasks/bin/copy_bin_scripts_task.ts

```diff
@@ -37,6 +37,7 @@ export const CopyBinScripts: Task = {
               darwin: platform.isMac(),
               linux: platform.isLinux(),
               serverless: platform.isServerless(),
+              overridePointerCompression: Boolean(process.env.CI_USE_POINTER_COMPRESSION), // for .buildkite/pipeline-resource-definitions/kibana-pointer-compression.yml
```

Comment:
`override` is fine just a small comment that maybe `force` is more in line with options we use on other CLIs

Reply:
something like `forceNodePointerCompression`

Reply:
a13e9b9b738bcc5c97d85dff456f251bf2e8aa12d